test/amf1-n2.yaml

```diff
@@ -25,7 +25,7 @@ spec:
         {
           "type": "macvlan",
           "capabilities": { "ips": true },
-          "master": "eth0",
+          "master": "eno1",
```

Comment:
Why the interface change?

Reply:
Interfaces have been changed just because of functional testing on our setup environment.